src/modules/private.ts

```diff
@@ -883,6 +885,43 @@ export default class Private {
     );
   }
 
+  /**
+   * @description get ethereum address restrictions on the dYdX protocol.
+   */
+  async getRestrictions(
+    genericParams: GenericParams = {},
+  ): Promise<{ restrictions: RestrictionResponseObject }> {
+    return this._get(
+      'restrictions',
+      {
+        ...genericParams,
+      },
+    );
+  }
+
+  /**
+   * @description comply to dYdX terms of service after a first offense.
+   */
+  async userCompliance(
```

Comment:
this is for external use right? Shouldn't we name this something that reflects the external api? Like `postRestrictionsCompliance`?

Reply:
done